### PR TITLE
{Pylint} Fix useless-object-inheritance

### PR DESCRIPTION
**Pull request by atombrella**
**Description**

This is redundant as of Python 3. This code base needs a bit of cleaning after the Python 3 only support.

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py [hunk 1]
```diff
@@ -1593,7 +1593,7 @@ def _update_login(server_name, resource_group_name, auth_config, password_auth,
 
 
 # pylint: disable=too-many-instance-attributes, too-few-public-methods, useless-object-inheritance
-class DbContext(object):
+class DbContext:
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py:1596]: [`useless-object-inheritance`](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/useless-object-inheritance.html) can be removed.

File: src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py [hunk 2]
```diff
@@ -1593,7 +1593,7 @@ def _update_login(server_name, resource_group_name, auth_config, password_auth,
 
 
 # pylint: disable=too-many-instance-attributes, too-few-public-methods, useless-object-inheritance
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py:1595]: [`useless-object-inheritance`](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/useless-object-inheritance.html) can be removed.

```suggestion
# pylint: disable=too-many-instance-attributes, too-few-public-methods
```

File: src/azure-cli/azure/cli/command_modules/mysql/custom.py [hunk 3]
```diff
@@ -1992,7 +1992,7 @@ def migrate_firewall_rules_from_single_to_flex(db_context, cmd, source_server_id
 
 
 # pylint: disable=too-many-instance-attributes, too-few-public-methods, useless-object-inheritance
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/mysql/custom.py:1994]:
```suggestion
# pylint: disable=too-many-instance-attributes, too-few-public-methods
```

Comment (atombrella) [reply src/azure-cli/azure/cli/command_modules/mysql/custom.py]: fixed

File: src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py (hunk 2 above)
Comment (atombrella) [reply src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py]: Fixed

**Review by jiasli — DISMISSED** [review]

**Comment by atombrella** [comment]
@jiasli I pushed another small related fix.

**Review by jiasli — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**